src/nodes.ts

```diff
@@ -22,31 +23,33 @@ export type PropertyKey = Identifier | Literal;
 export type PropertyValue = AssignmentPattern | BindingIdentifier | BindingPattern | FunctionExpression;
 export type StatementListItem = Declaration | Statement;
 
+type Nullable<T> = T | null;
```

Comment:
I don't have any strong preference on one or another, I'll revert it back to use `| null` again. Beside, I have the feeling that TypeScript will have `Nullable<T>` or `?T` at some point (which is proven to be quite popular in C# already).